### DISCUSSION TITLE
不知道怎么搞的正版登不上去

### DISCUSSION THREAD

**Discussion by laokaw** [Meloong-Git/PCL#2540] [提问] [answered]
[新建 文本文档.txt](https://github.com/Hex-Dragon/PCL2/files/12387715/default.txt)
前天还好好的昨天和今天就突然不行了皮肤也加载不出来

**Comment by zkitefly** [comment]
前往启动器设置将下载源更换

**Comment by hejiehao** [comment, marked as answer]
由于护网行动，中国移动、中国广电网络下部分 Microsoft 服务无法正常访问

自北京时间 2023 年 08 月 19 日 11 时起，包括必应搜索在内的部分 Microsoft 服务受到了部分地区中国移动省份网关屏蔽，流向 Microsoft 数据中心节点的数据包会在经过省份网关时被丢弃。

这是由于护网行动导致的，护网行动是以公安部牵头的，用以评估企事业单位的网络安全的活动。具体实践中，公安部会组织攻防两方，进攻方会在一个月内对防守方发动网络攻击，检测出防守方（企事业单位）存在的安全漏洞。通过与进攻方的对抗，企事业单位网络、系统以及设备等的安全能力会大大提高。

已知受影响地区有山东、河南、安徽、四川 、辽宁、陕西、湖北、浙江、湖南、重庆、江西、内蒙、青海。预计至九月能够恢复。

对于 Bing 搜索服务，你可以镜像网站 [https://s.cn.bing.net/](https://www.kookapp.cn/go-wild.html?url=https%3A%2F%2Fs.cn.bing.net%2F)
对于其他 Microsoft 服务，你可以选择更换运营商为中国连通、中国电信。或直接联系运营商客服解决。

解决方案可以查看本文： [https://answers.microsoft.com/zh-hans/microsoftedge/forum/msedge_issue-msedge_win11/%E5%85%B3%E4%BA%8E%E4%B8%AD%E5%9B%BD%E7%A7%BB/e0f15624-0873-4a88-9c06-e077b61e99a3](https://www.kookapp.cn/go-wild.html?url=https%3A%2F%2Fanswers.microsoft.com%2Fzh-hans%2Fmicrosoftedge%2Fforum%2Fmsedge_issue-msedge_win11%2F%25E5%2585%25B3%25E4%25BA%258E%25E4%25B8%25AD%25E5%259B%25BD%25E7%25A7%25BB%2Fe0f15624-0873-4a88-9c06-e077b61e99a3)
